org.opent2t.sample.lamp.superpopular/com.smartthings.lightbulb/js/thingTranslator.js

```diff
@@ -265,25 +265,24 @@ function resourceSchemaToProviderSchema(resourceId, resourceSchema) {
             }
             break
         default:
-            // Error case
-            console.log("Invalid resourceId: " + resourceId)
-            throw new Error("Invalid resourceId");
+        throw new Error("Invalid resourceId");
```

Comment:
Being fixed by Dan in his PR. Will eventually need to merge his changes into mine once he checks in.

---
In reply to: [103830413](https://github.com/openT2T/translators/pull/188#discussion_r103830413) [](ancestors = 103830413)